app/src/NativeBase/Components/Forms/ResponseModal.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * @file Modal shows response when a form is submitted.
+ */
+
+import React, { FC } from 'react'
+import { Text, Modal, Icon } from 'native-base'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+import StaTheme from '../../Theme/StaTheme'
+
+interface ResponseModalProps {
+  message: string
+  header: string
+  success: boolean
+  isOpen: boolean
+  onClose: (value: boolean) => void
+}
+
+const ResponseModal: FC<ResponseModalProps> = ({
+  message,
+  header,
+  success,
+  isOpen,
+  onClose,
+}) => {
+  if (success) {
+    return (
+      <Modal
+        isOpen={isOpen}
+        onClose={onClose}
+        justifyContent="center"
+        size="xl"
+      >
+        <Modal.Content
+          _dark={{ color: StaTheme.colors.bgDarkMode['100'] }}
```

Comment:
I've realised doing the NativeBase ticket I'm working on that there's a simpler (and more NativeBase) way to reference colours -- you can just do e.g. `_dark={{ color: 'bgDarkMode.100' }}` and similarly for the other colours below